docs/dapr-sdk-workflows/io/dapr/workflows/runtime/class-use/WorkflowActivity.html

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE HTML>
 <html lang="en">
 <head>
-<!-- Generated by javadoc (17) on Mon Dec 09 17:48:11 UTC 2024 -->
-<title>Uses of Interface io.dapr.workflows.runtime.WorkflowActivity (dapr-sdk-workflows 0.13.1 API)</title>
+<!-- Generated by javadoc (17) on Tue Aug 13 18:03:07 PDT 2024 -->
+<title>Uses of Interface io.dapr.workflows.runtime.WorkflowActivity (dapr-sdk-workflows 0.12.0 API)</title>
```

Comment:
Same. It is updated during release only.

Reply:
Done